nbsite/pyodide/__init__.py

```diff
@@ -129,8 +129,8 @@ def extract_extensions(code: str) -> List[str]:
         js_modules.update(design_resources['js_modules'])
     resources = Resources(mode='cdn')
     extensions = _bundle_extensions(None, resources)
-    js += [bundle.cdn_url for bundle in extensions if bundle.cdn_url and
-           '@holoviz/panel@' not in bundle.cdn_url]
+    js += [str(bundle.cdn_url) for bundle in extensions if bundle.cdn_url and
+           '@holoviz/panel@' not in str(bundle.cdn_url)]
```

Comment:
Just for fun:

```suggestion
    js += [cdn_url for bundle in extensions if bundle.cdn_url and
           '@holoviz/panel@' not in (cdn_url:= str(bundle.cdn_url))]
```

Reply:
I like where your head is. We likely need to also update this line to 3.8 then:

https://github.com/holoviz-dev/nbsite/blob/1a51ad3346dd6533ae71a60f424a9b6325cf6c19/setup.py#L55